dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -520,34 +523,33 @@ public BlockWriter createFileWriter(String fileId, String ufsPath)
     return new PagedFileWriter(this, ufsPath, mCacheManager, fileId, mPageSize);
   }
 
-  private boolean isAllPageCached(UfsFileStatus status, Optional<DoraMeta.FileStatus> fileStatus) {
-    if (!fileStatus.isPresent()) {
-      return false;
-    }
-    alluxio.grpc.FileInfo fi = fileStatus.get().getFileInfo();
-    if (!fi.getContentHash().equals(status.getContentHash())) {
-      return false;
+  private boolean isAllPageCached(Block block) {
+    alluxio.grpc.UfsStatus status = block.getUfsStatus();
+    String fileId = new AlluxioURI(status.getUfsFullPath()).hash();
```

Comment:
can u clean up all these places in pageddoraworker new AlluxioURI(ufsFullPath).hash()  to some common util places to generate file id?

Reply:
I would defer this to the PR replacing AlluxioURI with UFSURI